source/administration/object-management/object-retention.rst

```diff
@@ -234,7 +234,7 @@ the MinIO :mc:`mc` CLI, or using an S3-compatible SDK.
         MinIO deployment.
 
       - Replace ``BUCKET`` with the 
-        :mc-cmd:`name <mc version ALIAS>` of the bucket to create.
+        :mc-cmd:`name <mc version enable ALIAS>` of the bucket to create.
```

Comment:
I'm unclear what `mc version enable ALIAS` has to do with creating a bucket, but this made the warning shut up.

Reply:
Ah yeah this is just _wrong_. 

Should be

```suggestion
        :mc-cmd:`name <mc mb ALIAS>` of the bucket to create.
```

Reply:
changed along with `mc` vs `mc-cmd` fixup